app/Http/Controllers/AuthController.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use App\Models\User;
+
+/**
+ * 認証コントローラー
+ *
+ * ユーザーのログイン、登録、ログアウト処理を管理する
+ *
+ */
+class AuthController extends Controller
+{
+    /**
+     * ログインフォーム表示
+     *
+     * @return \Illuminate\Contracts\View\View
+     */
+    public function showLoginForm()
+    {
+        return view('auth.login');
+    }
+
+
+    /**
+     * ログイン処理
+     *
+     * @param Request $request リクエストオブジェクト
+     *
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function login(Request $request)
+    {
+        $credentials = $request->only('email', 'password');
+
+        if (Auth::attempt($credentials)) {
+            $request->session()->regenerate();
+            return redirect()->intended('/dashboard');
+        }
+
+        return back()->withErrors(['email' => 'メールアドレスまたはパスワードが正しくありません。']);
+    }
+
+    /**
+     * ユーザー登録処理
+     *
+     * @return \Illuminate\Contracts\View\View
+     */
+    public function showSignupForm()
+    {
+        return view('auth.signup');
+    }
+
+    /**
+     * ユーザー登録処理
+     *
+     * @param Request $request リクエストオブジェクト
+     *
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function register(Request $request)
```

Comment:
nitpick: 型の明記

return型を明記しておくと読みやすくなるよ！
Laravelでは推奨レベルらしい。

```suggestion
    public function register(Request $request): RedirectResponse
```

Reply:
@yu-saito-ceres 
修正しました
[268ba20](https://github.com/ceres-hackathon-2025/team-b/pull/4/commits/268ba2054f611cd1bd3ff0429baacd255623aa6b)
[f4c4798](https://github.com/ceres-hackathon-2025/team-b/pull/4/commits/f4c4798983e55cd4b547134bc1a3fd4fc0a54293)